db/seeds.rb

```diff
@@ -6,7 +6,7 @@
 # add config/service_providers.yml
 SERVICE_PROVIDERS.each do |issuer, config|
   ServiceProvider.find_or_create_by!(issuer: issuer) do |sp|
-    sp.approved = VALID_SERVICE_PROVIDERS.include?(issuer)
+    sp.approved = true
```

Comment:
the column mirrors what is in the dashboard, where at some point we may implement some kind of approval policy if we wanted to use it in more curated environments.